### ISSUE TITLE
I am getting error while running npm start

### ISSUE THREAD

**Issue by smanne** [kriasoft/react-starter-kit#263]
> @ start /Users/smanne/workspace/react/react-starter-kit
> babel-node --eval "require('./tools/start')().catch(err => console.error(err.stack))"

/Users/smanne/workspace/react/react-starter-kit/node_modules/babel/node_modules/babel-core/lib/transformation/file/plugin-manager.js:147
      throw new ReferenceError(messages.get("pluginUnknown", name));
            ^
ReferenceError: Unknown plugin "react-transform"
    at PluginManager.subnormaliseString (/Users/smanne/workspace/react/react-starter-kit/node_modules/babel/node_modules/babel-core/lib/transformation/file/plugin-manager.js:147:13)
    at PluginManager.add (/Users/smanne/workspace/react/react-starter-kit/node_modules/babel/node_modules/babel-core/lib/transformation/file/plugin-manager.js:190:40)
    at File.buildTransformers (/Users/smanne/workspace/react/react-starter-kit/node_modules/babel/node_modules/babel-core/lib/transformation/file/index.js:237:21)
    at new File (/Users/smanne/workspace/react/react-starter-kit/node_modules/babel/node_modules/babel-core/lib/transformation/file/index.js:139:10)
    at Pipeline.transform (/Users/smanne/workspace/react/react-starter-kit/node_modules/babel/node_modules/babel-core/lib/transformation/pipeline.js:164:16)
    at _eval (/Users/smanne/workspace/react/react-starter-kit/node_modules/babel/lib/_babel-node.js:78:16)
    at Object.<anonymous> (/Users/smanne/workspace/react/react-starter-kit/node_modules/babel/lib/_babel-node.js:106:16)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)

npm ERR! Darwin 14.4.0
npm ERR! argv "node" "/usr/local/bin/npm" "start"
npm ERR! node v0.12.2
npm ERR! npm  v2.7.5
npm ERR! code ELIFECYCLE
npm ERR! @ start: `babel-node --eval "require('./tools/start')().catch(err => console.error(err.stack))"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ start script 'babel-node --eval "require('./tools/start')().catch(err => console.error(err.stack))"'.
npm ERR! This is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     babel-node --eval "require('./tools/start')().catch(err => console.error(err.stack))"
npm ERR! You can get their info via:
npm ERR!     npm owner ls
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/smanne/workspace/react/react-starter-kit/npm-debug.log

**Comment by loljoho** [comment]
First forage into React and same problem!

Went out on a limb and tried `npm install -g babel` and I got a little further with `npm start`:

```
> @ start /media/joho/www/sandbox-react
> babel-node --eval "require('./tools/start')().catch(err => console.error(err.stack))"

build
clean
copy
build/package.json
bundle
Time: 51233ms
        Asset    Size  Chunks             Chunk Names
    server.js  182 kB       0  [emitted]  main
server.js.map  108 kB       0  [emitted]  main
Time: 56155ms
     Asset     Size  Chunks             Chunk Names
    app.js  1.16 MB       0  [emitted]  main
app.js.map  1.29 MB       0  [emitted]  main
serve
The server is running at http://localhost:5000
[BS] Proxying: http://localhost:5000
[BS] Access URLs:
 --------------------------------------
       Local: http://localhost:3000
    External: http://192.168.20.11:3000
 --------------------------------------
          UI: http://localhost:3001
 UI External: http://192.168.20.11:3001
 --------------------------------------
child_process.js:1162
    throw errnoException(err, 'spawn');
          ^
Error: spawn EACCES
...
...
```

I also took a glance at [this page](https://babeljs.io/docs/usage/cli/) and attempted to use `babel` on the CLI to compile the `tools/` and go from there, but `npm start` threw me an `ENOSPC` error immediately after `build/package.json`, and `sudo npm start` gave me the same as above.

I'm a total newbie to most of the components in this setup FYI, and I'm sure it has something to do with whichever current version(s) we're trying to use...  Hope this post helps someone out.

Cheers :)

**Comment by brandon-arnold** [comment]
Not sure why these issues are being ignored. Fixed in PR #272. Add a line to your package.json in the meantime:

```javascript
    "babel-core": "^5.4.0",
```

**Comment by web-jenezis** [comment]
Hi @brandon-arnold,
I'm got some errors yesterday, here is the errors:
```
$ npm run build

> @ build /var/www/public/jenezis
> babel-node tools/run build

[20:37:07] Starting 'build'...
[20:37:07] Starting 'clean'...
[20:37:08] Finished 'clean' after 236 ms
[20:37:08] Starting 'copy'...
build/package.json
[20:37:08] Finished 'copy' after 128 ms
[20:37:08] Starting 'bundle'...
Child
    Time: 7968ms

    ERROR in ./src/client.js
    Module build failed: ReferenceError: [BABEL] /var/www/public/jenezis/src/client.js: Unknown option: /var/www/public/jenezis/.babelrc.presets
        at Logger.error (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/file/logger.js:58:11)
        at OptionManager.mergeOptions (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/file/options/option-manager.js:126:29)
        at OptionManager.addConfig (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/file/options/option-manager.js:107:10)
        at OptionManager.findConfigs (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/file/options/option-manager.js:168:35)
        at OptionManager.init (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/file/options/option-manager.js:229:12)
        at File.initOptions (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/file/index.js:147:75)
        at new File (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/file/index.js:137:22)
        at Pipeline.transform (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/pipeline.js:164:16)
        at transpile (/var/www/public/jenezis/node_modules/babel-loader/index.js:14:22)
        at Object.module.exports (/var/www/public/jenezis/node_modules/babel-loader/index.js:88:12)
Child
    Time: 7840ms

    ERROR in ./src/server.js
    Module build failed: ReferenceError: [BABEL] /var/www/public/jenezis/src/server.js: Unknown option: /var/www/public/jenezis/.babelrc.presets
        at Logger.error (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/file/logger.js:58:11)
        at OptionManager.mergeOptions (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/file/options/option-manager.js:126:29)
        at OptionManager.addConfig (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/file/options/option-manager.js:107:10)
        at OptionManager.findConfigs (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/file/options/option-manager.js:168:35)
        at OptionManager.init (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/file/options/option-manager.js:229:12)
        at File.initOptions (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/file/index.js:147:75)
        at new File (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/file/index.js:137:22)
        at Pipeline.transform (/var/www/public/jenezis/node_modules/babel-core/lib/transformation/pipeline.js:164:16)
        at transpile (/var/www/public/jenezis/node_modules/babel-loader/index.js:14:22)
        at Object.module.exports (/var/www/public/jenezis/node_modules/babel-loader/index.js:88:12)
[20:37:16] Finished 'bundle' after 8717 ms
[20:37:16] Finished 'build' after 9088 ms
```

I fixed the problem with babel update. I added the string `"babel-core": "latest",` to `package.json` and problem was fixed.
